setup.py

```diff
@@ -255,7 +255,9 @@ def get_torch_arch_list() -> Set[str]:
 ]
 
 if _is_cuda():
-    vllm_extension_sources.append("csrc/quantization/awq/gemm_kernels.cu")
+    vllm_extension_sources.extend([
```

Comment:
What is the purpose of this change?

Reply:
This is an artifact of an earlier commit in my branch. Will fix

Reply:
Fixed!